project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/in/action.less

```diff
@@ -31,14 +21,9 @@
 
         @media @query-md {
             order: 2;
-            flex: 1;
+            width: 100%/2;
```

Comment:
Same as the comment above.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width same for both 50%.

Reply:
It is okay for now. We will refactor this later.